addons/beehave/nodes/beehave_node.gd

```diff
@@ -4,6 +4,11 @@ class_name BeehaveNode extends BeehaveTree
 @icon("../icons/action.svg")
 
 ## Executes this node and returns a status code.
-## This method must be overwritten. 
+## This method must be overwritten.
 func tick(actor: Node, blackboard: Blackboard) -> int:
 	return SUCCESS
+
+
+## Called when this node needs to be interrupted before it can return FAILURE or SUCCESS.
+func interrupt(actor: Node, blackboard: Blackboard) -> void:
```

Comment:
another thought: any reason this cannot be a signal instead?

`
signal interrupted(actor: Node, blackboard: Blackboard)
`

Reply:
I would use a signal if it was trying to call a parent from a child node but in this case its the opposite. As a rule of thumb, I use `parent -> child = method` and `child -> parent = signal`.